crates/fuel-core/src/schema/dap.rs

```diff
@@ -295,25 +322,13 @@ impl DapMutation {
         Ok(result)
     }
 
-    #[cfg(not(feature = "debug"))]
-    async fn set_single_stepping(
-        &self,
-        _ctx: &Context<'_>,
-        _id: ID,
-        _enable: bool,
-    ) -> async_graphql::Result<bool> {
-        Err(async_graphql::Error::new(
-            "Feature 'debug' is not compiled in",
-        ))
-    }
-
-    #[cfg(feature = "debug")]
     async fn set_single_stepping(
         &self,
         ctx: &Context<'_>,
         id: ID,
         enable: bool,
     ) -> async_graphql::Result<bool> {
+        require_debug(ctx)?;
```

Comment:
Where before  `#[cfg(feature = "debug")]` prevented these from showing up in the api at all, will this `require_debug` approach mean that `set_single_stepping` (and the other endpoints) will still be available to non-debug apis, but will always error?

Reply:
Yes, it is what we discussed=) So it is totally expected behavior

Reply:
I'm kinda uncomfortable with exposing api endpoints to the standard user that they can never hit. But it's not a huge concern, it just seems messy.